test/test-cli.py

```diff
@@ -242,7 +242,7 @@ def test_config_error_for_multiple_js_files_is_printed_only_once(self) -> None:
                 encoding="utf-8",
             )
             self.assertEqual(result.returncode, 1)
-            self.assertEqual(result.stderr.count("E0164"), 1)
+            self.assertEqual(result.stderr.count("E0164"), 2)
```

Comment:
This is confusing. We should instead do something like strip escape sequences in the test.

Must fix: Actually, the CLI shouldn't use escape sequences if the output is a pipe. We should only print escape sequences if the output is a terminal. (You can check if the output is a terminal using `isatty` on POSIX platforms.)